src/urbit/app/chess.hoon

```diff
@@ -301,14 +364,7 @@
             :~  [%give %poke-ack `~[leaf+err]]
             ==
           =,  u.new.move-result
-          :-  :_  cards.move-result
-              ::  send move to opponent
-              :*  %give
-                  %fact
-                  ~[/game/(scot %da game-id.action)/moves]
-                  %chess-move
-                  !>(move.action)
-              ==
+          :-  cards.move-result
```

Comment:
I moved the move card to `try-move` in order to avoid sending an update on the `/moves` wire when we're sending a `%chess-game-result`, which includes the move if needed to verify that a special draw is claimable. But I think you're right that my version is going to send the move whether we're making a move or verifying an opponent's move. Is it a problem if we end up sending two updates on the `/moves` wire—the move as well as a `%chess-game-result`? In [this comment](https://github.com/thecommons-urbit/chess/pull/40#pullrequestreview-1121057259) you mentioned that would technically break the logic of the draw claim being sent with a move. I guess the alternative is to add a check in this arm whether we're auto-claiming a special draw so that we don't send the two separate cards.

Reply:
Ah, yeah, I see - you're right. I guess we add another `ship-to-move` copy into `try-move` and use it to determine which cards to send? This'll be ugly but it's not so bad, since #44 also adds `ship-to-move` into `try-move` if I remember correctly.